client/src/components/dashboard/OverviewPage.jsx

```diff
@@ -162,17 +162,18 @@ class OverviewPage extends Component {
 
   render() {
     const { classes, interviews, user } = this.props;
+    const { scheduledCandidateIds } = this.state;
     const { profile } = user;
 
     return (
       <div>
         {
           profile
-            && (
-              <h1 className={classes.title}>
-                Welcome, {profile.givenName}
-              </h1>
-            )
+          && (
```

Comment:
Pretty sure according to airbnb this should be indented...

Reply:
ohh I don't know, this was done by VScode..